include/deal.II/lac/utilities.h

```diff
@@ -445,11 +445,11 @@ namespace Utilities
       // algorithm verbatim:
       //
       // [Y]=chebyshev_filter_scaled(X, m, a, b, aL).
-      // e=(b−a)/2; c=(a+b)/2; σ=e/(c−aL); τ=2/σ;
-      // Y=(H∗X−c∗X)∗(σ/e);
+      // e=(b-a)/2; c=(a+b)/2; σ=e/(c-aL); τ=2/σ;
+      // Y=(H∗X-c∗X)∗(σ/e);
       // for i=2 to m do
-      //   σnew =1/(τ −σ);
-      //   Yt =(H∗Y−c∗Y)∗(2∗σnew/e)−(σ∗σnew)∗X;
+      //   σnew =1/(τ - σ);
+      //   Yt =(H∗Y - c∗Y)∗(2∗σnew/e)-(σ∗σnew)∗X;
       //   X =Y; Y =Yt; σ =σnew;
```

Comment:
Are these other symbols `τ`, `σ`, etc. actually ascii?

Reply:
No, but since its not in math mode I left it alone. I suppose I am being slightly inconsistent in my removal of non-ascii characters, but I wanted to consistently remove the minus sign.

Reply:
Didn't think so :-D But its certainly not important to clean this up, so we won't worry about this now.